test/common/upstream/health_check_fuzz.cc

```diff
@@ -149,8 +149,12 @@ void HealthCheckFuzz::raiseEvent(const test::common::upstream::RaiseEvent& event
 
 void HealthCheckFuzz::replay(const test::common::upstream::HealthCheckTestCase& input) {
   for (int i = 0; i < input.actions().size(); ++i) {
+    if (i == 64) {
```

Comment:
you can encode this limit in the for loop (if i is less than the minimum of action size and some constant MaxActions, see style guides or examples for how to name/declare constants).

Reply:
What was the name of the fuzz test example you gave again in standup? The one where you guys capped at 1024 I think it was?

Reply:
codec_impl_fuzz_test, buffer_fuzz.cc, header_map_impl_fuzz_test all have this pattern!

Reply:
Thanks so much!